### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
##### Checklist

- [X] Documentation is added or updated
- [ ] New files have a LGPL 2.1 license statement
- [ ] PKCS#11 module is tested
- [ ] Windows minidriver is tested
- [ ] macOS tokend is tested

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -7,7 +7,7 @@ NEWS for OpenSC -- History of user visible changes
  * Add threading test to `pkcs11-tool` (#2067)
  * Add support to generate generic secret keys (#2140)
  * `opensc-explorer`: Print information about LCS (Life cycle status byte) (#2195)
- * Add support for Apple's arm64 (M1) binaries, removed TokenD. A seperate installer with TokenD (and without arm64 binaries) will be available (#2179).
+ * Add support for Apple's arm64 (M1) binaries, removed TokenD. A separate installer with TokenD (and without arm64 binaries) will be available (#2179).
```
Comment (Jakuje) [inline NEWS:10]: I think the `NEWS` file should be append only. I was already turned down with one typo fix in there, but lest see what others will think.

**Review by Jakuje — APPROVED** [review]

File: NEWS (hunk 1 above)
Comment (DimitriPapadopoulos) [reply NEWS:10]: OK, I can revert that if needed. I can understand NEWS is a pristine copy of past NEWS, with their typos. Wait and see?

**Review by frankmorgner — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
I could add a codespell GitHub Action if you're interested - in another PR.

**Comment by Jakuje** [comment]
I am all for having this in action in OpenSC. But I am sad that github still does not support the third result of the actions "warning" that would not make the whole check fail, but it would be simply to distinguish if there is an issue or not.

How does the action work? Does it check all code base or files changed in the PR?

@frankmorgner what about the the NEWS file? Last time I tried to [fix](https://github.com/OpenSC/OpenSC/pull/1960#discussion_r530191835) something in there, it was turned down.

**Comment by DimitriPapadopoulos** [comment]
I think the action checks the whole code base. I haven't looked into the details.

**Comment by frankmorgner** [comment]
here's what we *should* have in NEWS... https://www.gnu.org/prep/standards/html_node/NEWS-File.html

Looking back at this today, I think we should be fine correcting typos...

**Comment by Jakuje** [comment]
> I think the action checks the whole code base. I haven't looked into the details.

That would mean that we would have to have a lot of "exceptions" in dictionary to have some useful reports in the PRs, which I am not sure how viable is. I don't have time to experiment with it right now, but if you could draft a something that will work, I think we can make use of it.

**Comment by DimitriPapadopoulos** [comment]
I'll do that in a different PR.

**Comment by Jakuje** [comment]
> I'll do that in a different PR. Is that OK?

Yes please.

I merged this one as all the outstanding comments are resolved. Thank you!

**Pull request merged**